centaur/src/main/scala/centaur/CromwellTracker.scala

```diff
@@ -0,0 +1,60 @@
+package centaur
+
+import centaur.CromwellTracker._
+import cromwell.api.model.WorkflowMetadata
+import cromwell.core.{WorkflowProcessingEvent, WorkflowProcessingEvents}
+import cromwell.core.WorkflowProcessingEvents.DescriptionEventValue
+import io.circe.generic.semiauto._
+import io.circe.parser.parse
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+import org.apache.commons.math3.stat.inference.ChiSquareTest
+import org.slf4j.LoggerFactory
+
+import scala.language.postfixOps
+
+
+case class CromwellTracker(backendCount: Int, configuredSignificance: Double) {
+  val log = LoggerFactory.getLogger(getClass)
+  var counts: Map[String, Int] = Map()
+  def track(metadata: WorkflowMetadata): Unit = {
+
+    // Get all the workflow processing events for this workflow and pick out the "PickedUp" events. There can be multiple of
+    // these for a workflow if Cromwell was restarted while it was running.
+    val pickupEvents = parse(metadata.value) flatMap { _.hcursor.get[Array[WorkflowProcessingEvent]](WorkflowProcessingEvents.ProcessingEventsKey) }
```

Comment:
ToL-ish: I find [Circe Optics](https://circe.github.io/circe/optics.html) just-as-dense-to-read-as-HCursors while being prettier. I don't really know what the other tradeoffs would be. Here's what I think the optics version would look like.

```suggestion
    val pickupEvents = parse(metadata.value) map {
      root.workflowProcessingEvents.each.as[WorkflowProcessingEvent].getAll(_).toArray
    }
```

Note: Would also require `import io.circe.optics.JsonPath._` and later a `implicit lazy val workflowProcessingEventEncoder: Encoder[WorkflowProcessingEvent] = deriveEncoder` as optics support round-trips.